flake8_brackets_position/__init__.py

```diff
@@ -0,0 +1,179 @@
+            elif symb == ")" or symb == "(" or symb == "]" or symb == "[" or symb == "}" or symb == "{":
+                return True
+            else:
+                return False
+
+    def _adges_of_node(self, node):
+        end_lineno = 0
+        end_col_offset = 0
+        for child in ast.iter_child_nodes(node):
+            end_lineno_, end_col_offset_ = self._adges_of_node(child)
+            if end_lineno_ > end_lineno:
+                end_lineno, end_col_offset = end_lineno_, end_col_offset_
+            try:
+                if child.lineno > end_lineno:
+                    end_lineno = child.lineno
+                    end_col_offset = child.col_offset
+            except AttributeError:
+                continue
+        return end_lineno, end_col_offset
+
+    def check_parentheses_in_funcs(self) -> None:
+        exeption = []
+        for node in ast.walk(self.tree):
+            for cords in self.recheck_list:
+                try:
+                    if cords[0][0] == node.lineno and cords not in exeption:
+                        end_lineno, end_col_offset = self._adges_of_node(node)
+                        if end_lineno - node.lineno > 0 and not (node.lineno, node.col_offset) > cords[0]:
+                            if (self._last_in_line(cords[0], self.source_code_by_lines)
+                                    and not self._first_in_line(cords[3], self.source_code_by_lines)):
+                                self.problems.append((
+                                    cords[0][0], cords[0][1],
+                                    "BRA001: Opening bracket is last, but closing is not on new line"
+                                ))
+                                continue
+                            if (self._last_in_line(cords[0], self.source_code_by_lines)
+                                    and self._first_in_line(cords[3], self.source_code_by_lines)
+                                    and node.col_offset != cords[3][1]):
+                                self.problems.append((
+                                    cords[0][0], cords[0][1],
+                                    "BRA002: Closing is on new line but indentation mismatch"
+                                ))
+                                continue
+                            exeption.append(cords)
+
+                except AttributeError:
+                    continue
+
+
+OP_TOKEN_CODE = 54 if sys.version_info >= (3, 8) else 53
+
+
+def find_parens_coords(token):
+    # return parentheses paris in the form
+    # (
+    #   (open_line, open_col),
+    #   open_end_col,
+    #   replacement,
+    #   (close_line, close_col)
+    # )
+    open_list = ["[", "{", "("]
+    close_list = ["]", "}", ")"]
+    opening_stack = []
+    parentheses_pairs = []
+    last_line = -1
+    for i in range(len(token)):
+        first_in_line = last_line != token[i].start[0]
+        last_line = token[i].end[0]
+        if token[i].type == OP_TOKEN_CODE:
```

Comment:
```suggestion
        if token[i].type == tokenize.OP:
```

you then can remove the line above where we hard-code the number for each Python version.